src/flb_http_client.c

```diff
@@ -1483,6 +1500,19 @@ int flb_http_get_response_data(struct flb_http_client *c, size_t bytes_consumed)
             }
             available = flb_http_buffer_available(c) - 1;
         }
+        time_t now = time(NULL);
```

Comment:
Nits: Do we need to define this variable on top of this function?
Just for confirming to following our coding guideline.